Peer Dependency Warning when installing project created with npm create vue@latest

I encountered peer dependency warnings related to vite when creating a new Vue project using npm create vue@latest. This occurred during the installation of vite-plugin-inspect as a dependency of vite-plugin-vue-devtools.

Here are the warning messages:

```bash
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: vite-plugin-inspect@0.8.8
npm WARN Found: vite@6.0.2
npm WARN node_modules/vite
npm WARN   peer vite@"^5.0.0 || ^6.0.0" from @vitejs/plugin-vue@5.2.1
npm WARN   node_modules/@vitejs/plugin-vue
npm WARN     dev @vitejs/plugin-vue@"^5.2.1" from the root project
npm WARN   4 more (vite-hot-client, vite-plugin-vue-devtools, ...)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer vite@"^3.1.0 || ^4.0.0 || ^5.0.0-0" from vite-plugin-inspect@0.8.8
npm WARN node_modules/vite-plugin-vue-devtools/node_modules/vite-plugin-inspect
npm WARN   vite-plugin-inspect@"0.8.8" from vite-plugin-vue-devtools@7.6.7
npm WARN   node_modules/vite-plugin-vue-devtools
npm WARN
npm WARN Conflicting peer dependency: vite@5.4.11
npm WARN node_modules/vite
npm WARN   peer vite@"^3.1.0 || ^4.0.0 || ^5.0.0-0" from vite-plugin-inspect@0.8.8
npm WARN   node_modules/vite-plugin-vue-devtools/node_modules/vite-plugin-inspect
npm WARN     vite-plugin-inspect@"0.8.8" from vite-plugin-vue-devtools@7.6.7
npm WARN     node_modules/vite-plugin-vue-devtools
```

**Steps to Reproduce:**

1. Run `npm create vue@latest`.
2. Select options to scaffold a new Vue 3 project with Vite.
3. Observe the warning messages during dependency installation.

--- a/packages/playground/applet/package.json
+++ b/packages/playground/applet/package.json
@@ -28,7 +28,7 @@
     "typescript": "^5.7.2",
     "unocss": "^0.64.1",
     "vite": "^6.0.1",
-    "vite-plugin-inspect": "0.8.8",
+    "vite-plugin-inspect": "0.8.9",
     "vite-plugin-vue-devtools": "workspace:*"
   }
 }

--- a/packages/playground/basic/package.json
+++ b/packages/playground/basic/package.json
@@ -27,7 +27,7 @@
     "typescript": "^5.7.2",
     "unocss": "^0.64.1",
     "vite": "^6.0.1",
-    "vite-plugin-inspect": "0.8.8",
+    "vite-plugin-inspect": "0.8.9",
     "vite-plugin-vue-devtools": "workspace:*"
   }
 }

--- a/packages/playground/multi-app/package.json
+++ b/packages/playground/multi-app/package.json
@@ -20,7 +20,7 @@
     "typescript": "^5.7.2",
     "unocss": "^0.64.1",
     "vite": "^6.0.1",
-    "vite-plugin-inspect": "0.8.8",
+    "vite-plugin-inspect": "0.8.9",
     "vite-plugin-vue-devtools": "workspace:*"
   }
 }

--- a/packages/vite/package.json
+++ b/packages/vite/package.json
@@ -55,7 +55,7 @@
     "@vue/devtools-shared": "workspace:^",
     "execa": "^9.5.1",
     "sirv": "^3.0.0",
-    "vite-plugin-inspect": "0.8.8",
+    "vite-plugin-inspect": "0.8.9",
     "vite-plugin-vue-inspector": "^5.3.1"
   },
   "devDependencies": {

